pkg/vpcmodel/grouping.go

```diff
@@ -219,25 +219,41 @@ func (g *groupingConnections) addPublicConnectivity(ep EndpointElem, commonProps
 	(*g)[ep][connKey].appendNode(targetNode)
 }
 
-// vsiGroupingBySubnets returns a slice of EndpointElem objects, by grouping set of elements that
+// vsiOrSubnetsGroupingBySubnetsOrVsis returns a slice of EndpointElem objects, by grouping set of elements that
+// 1. If the grouped elements are vsis then the elements
 // represent network interface nodes from the same subnet into a single groupedNetworkInterfaces object
```

Comment:
> into a single groupedNetworkInterfaces object

what is `groupedNetworkInterfaces ` object? should update the comment here

Reply:
it was a typo (not from this PR), fixed